controller/controller-board/src/app/controllers/TaskController.js

```diff
@@ -86,7 +86,7 @@
             taskService.getTasks(vm.namespace, vm.clustername, vm.state)
                 .then(function(result) {
                     vm.taskStatusCode = result.status;
-                    vm.tasks = result.data;
+                    vm.tasks = result.data.reverse();
```

Comment:
a simple reverse is good enough?

Reply:
yes, I noticed that the task ids are indeed incremented when there is a new task coming in and the bug is actually introduced by the $index from angular. So I removed them. 

However I don't know what will happen when we exhaust all the id numbers, so we should still use the actual date to sort, however, this makes the rendering time longer.

Reply:
good point using dates. 
the id probably won't overflow anytime soon, but let's use dates.